packages/lib-sourcify/package.json

```diff
@@ -18,7 +18,7 @@
     "check": "run-s check:*",
     "check:eslint": "eslint . --ext .ts",
     "check:prettier": "prettier \"./**/*.ts\" --check",
-    "test": "c8 --reporter=none mocha -r ts-node/register test/**/*.spec.ts --no-timeout --exit",
+    "test": "c8 --reporter=none mocha -r ts-node/register test/**/*.spec.ts test/*.spec.ts --no-timeout --exit",
```

Comment:
Why do you need this? shouldn't `test/*.spec.ts` be already included in the superset `test/**/*.spec.ts`?

Reply:
Yes, but for some reason it didn't do it for me 🤷

Reply:
Yes